fury/actor/curved.py

```diff
@@ -810,12 +1110,80 @@ def streamtube(
         Material model (e.g., 'phong', 'basic').
     enable_picking : bool, optional
         If True, the actor can be picked in a 3D scene.
+    backend : {"cpu", "gpu", None}, optional
+        Backend selection for streamtube generation. Options:
+        - ``"cpu"``: Force CPU-based geometry generation (default fallback).
+        - ``"gpu"``: Use GPU compute shaders for geometry generation.
+        - ``None``: Automatically choose GPU baked workflow when available.
```

Comment:
Rather than having `None` as one of the option please have option called `gpu_backed`, and explain it instead of None.

If they explicitly send None => Provide an info log saying switching to gpu_backed.

Reply:
But, I agree None here does not add much